README.md

```diff
@@ -135,9 +132,26 @@ Your script needs to have only one argument and to be a wrapper for a Terraform
 * Usage
 
 ```
-# Check terraform script parameters
+# Check script parameters
+source ./bash-methods/terraform/get-script-parameters.sh
+```
+
+or
+
+```
+# Check custom script parameters
+export SCRIPT_PARAMS_VARIABLES=(PROVIDER REGION STEP)
+export SCRIPT_PARAMS_USAGE='<provider> <region> <step ex.init|plan|apply>'
 source ./bash-methods/terraform/get-script-parameters.sh
 ```
+### Get parameters for Terraform script
```

Comment:
why not in the `Terraform` part? And also add Requirements and Usage for that, it is better to repeat things in the documentation instead of not be clear I think